tutorials/dataframe/df107_SingleTopAnalysis.py

```diff
@@ -0,0 +1,245 @@
+## \file
+## \ingroup tutorial_dataframe
+## \notebook -draw
+## \brief A single top analysis using the ATLAS Open Data release of 2020, with RDataFrame.
+##
+## This tutorial is the analysis of single top production adapted from the ATLAS Open Data release in 2020
+## (http://opendata.atlas.cern/release/2020/documentation/). The data was recorded with the ATLAS detector
+## during 2016 at a center-of-mass energy of 13 TeV. Top quarks with a mass of about 172 GeV are preliminary
+## produced in pairs but also appear alone, dominantly from the decays of a W boson in association with a light jet.
+##
+## The analysis is translated to a RDataFrame workflow processing a subset of 60 GB of simulated events and data.
+## The original analysis is reduced by a factor given as the command line argument --lumi-scale.
+##
+## \macro_image
```

Comment:
Can you upload the image to our webserver, and link to it? If not, the doxygen job (which takes 8h already) has to rerun the whole thing every night. Same for macro_output ...

Reply:
I think we want to have it computed to see that everything is still in place, no?